usr/bin/NETMODE

```diff
@@ -1,34 +1,39 @@
-#!/bin/bash
+#!/bin/sh
 
-function show_usage() {
+show_usage() {
     echo "Usage: $0 [DHCP_CLIENT|DHCP_SERVER|AUTO]"
     echo ""
 }
 
-function shut_it_all_down() {
+shut_it_all_down() {
     #we are restarting, possibly with new config, or reversing dhcp roles
```

Comment:
Please drop comments

Reply:
comments don't slow the code and explain to others the intent of the code. on principle, no.

Reply:
I know comments don't slow execution. The function is simple enough to not need a comment.

Reply:
my concern was that this function looks like it does too much if you don't consider all the options (like switching mode and coming from an unknown state). I wanted it clear why the function was so kill happy